### ISSUE TITLE
Preview my work in real time

### ISSUE THREAD

**Issue by tomcrane** [digirati-co-uk/iiif-manifest-editor#19]
- I would like to be able to preview the manifest I am working on, as I go.
- I'd like to see what it looks like in popular viewers such as UV and Mirador.
- I'd like to do this quickly and naturally as part of my workflow, rather than pause, publish to the web, wait a while... - the preview needs to be immediate and not conditional on me committing to "publish" my manifest anywhere.

**Comment by tomcrane** [comment]
Implementation issues - https://github.com/digirati-co-uk/iiif-manifest-editor/issues/12, #1

**Comment by am2539** [comment]
Seems very sensible -- especially when it comes to metadata editing. This would necessitate that the manifest be always resolvable (not a problem as long as relevant bits are escaped etc.), I'm sure you've thought of this already.

**Comment by sammeltassen** [comment]
If the manifest is saved locally in the Manifest Editor (using Filesystem api) it could be loaded via string queries in the viewers if there's a local web server running (?). How does [pdf.js](https://github.com/mozilla/pdf.js) allow for opening local files? By uploading to the browser? (But this would require changes on the viewer side.)

**Comment by tomcrane** [comment]
See https://github.com/digirati-co-uk/iiif-manifest-editor/blob/main/docs/rfcs/001-iiif-sandbox.md for our lowest-common-denominator approach. 

Also see https://github.com/digirati-co-uk/iiif-manifest-editor/wiki/Preview for how other mechanisms might be configured.

```
  {
    "label": "Ocean Liners",
    "mechanism": "previewService",
    "template": "https://canvas-panel.digirati.com/#/examples/fullpage?manifest={iiifResource}" 
  },
```

**Comment by tomcrane** [comment]
Addressed by #26
